pages/profile/views/index.jsx

```diff
@@ -47,7 +47,38 @@ class Index extends React.Component {
           </Fragment>
         }
       </dl>
-
+      <ExpandingPanel title={<Snippet>pil.training.title</Snippet>}>
+        {
+          model.certificates && model.certificates.length > 0
+            ? model.certificates.map((certificate, index) => (
+              <div key={index}>
+                <p><Snippet>pil.training.certificate.number</Snippet><span>:</span> {certificate.certificateNumber}</p>
+                <p><Snippet>pil.training.certificate.awarded</Snippet><span>:</span> {dateFormatter(certificate.passDate)}</p>
+                <p><Snippet>pil.training.certificate.body</Snippet><span>:</span> {certificate.accreditingBody}</p>
+                <p><Snippet>pil.training.certificate.modules</Snippet></p>
+                <p><ul>
+                  { certificate.modules.map((module, index) => (
+                    <Fragment key={index}>
+                      <li>{module.module}</li>
+                      {
+                        module.species && !!module.species.length && (
+                          <ul>
+                            {
+                              module.species.map((s, index) =>
+                                <li key={index}>{s}</li>
+                              )
+                            }
+                          </ul>
+                        )
+                      }
+                    </Fragment>
+                  )) }
+                </ul></p>
+              </div>
+            ))
+            : <p><em><Snippet>pil.training.none</Snippet></em></p>
+        }
+      </ExpandingPanel>
```

Comment:
This looks familiar, if the same code is being used in multiple places could we make it a component and import it here?

Reply:
done.